src/main/steps/index.ts

```diff
@@ -52,6 +52,8 @@ const getNextIncompleteStep = (
 
       return nextStep
         ? getNextIncompleteStep(data, nextStep, sequence, removeExcluded, checkedSteps.concat(step))
+        : [State.ConditionalOrderDrafted, State.ConditionalOrderPending].includes(data.state)
```

Comment:
i think this needs to be its own if/case statement.

Reply:
as in split that return statement into multiple if/else or that the check for that should be separate from the nextStep check? 

I thought it was worth keeping it in the nextStep check to make sure the continue-with-your-application question had been answered

Reply:
just make it 1 if/else statement as i feel like the inline condition should really be used for if or else but when theres multiple conditions it should just be its own if (nextstep) else if (this state) else ....